app/models/reachLimit.ts

```diff
@@ -0,0 +1,52 @@
+import { ObjectId } from "mongodb";
+import { database } from "./db";
+
+export enum CacheTimes {
+  None = 0,
+  Second = 1000,
+  Minute = 60 * 1000,
+  Hour = CacheTimes.Minute * 60 * 1000,
+  Day = CacheTimes.Hour * 24 * 1000,
+  Month = CacheTimes.Day * 30 * 1000,
+  Year = CacheTimes.Day * 365 * 1000,
+}
+
+interface ReachLimit {
+  _id: ObjectId;
+  ip: string;
+  address: string;
+  expireTime: number;
+}
+
+export const reachLimitCollection =
+  database.collection<ReachLimit>("reachlimits");
+
+export const setReachLimit = async (ip: string, address: string) => {
+  await reachLimitCollection.updateOne(
+    { ip, address },
+    { $set: { expireTime: Date.now() + CacheTimes.Day } },
```

Comment:
Could we config interval?

Reply:
yes kub, I will add config to env file kub